### ISSUE TITLE
Bug: The map I'm working on has tons of missing turfs despite it being fine last night

### ISSUE THREAD

**Issue by ophaq** [SpaiR/StrongDMM#160]
### Version

2.6.2 alpha

### What happened?

Last night I was adding objects/furnishings to my map after finally getting decals and such done. I load up SDMM today and now my map appears as a corrupted mess. There are tons of tiles missing everywhere, tiles where they shouldn't be, etc. I don't know how or why this happened because I didn't do anything but edit my map.

I tried the map in a local test server and the changes I made appear fine in-game. I tried an older version of the map I had saved before I did the furnishings and it looks fine in SDMM. I'm assuming something/some object in SDMM is causing my map in its current state to not appear right or corrupted.

### Reproduction

1. Run StrongDMM
2. Try this map in it: https://github.com/ophaq/testing/tree/scptestingmap-1/_maps/map_files/TestSCPMap_2Small500
3. It appears as a corrupted mess.
4. Attached my recent log below
[2022.08.05-22.29.32.log](https://github.com/SpaiR/StrongDMM/files/9274331/2022.08.05-22.29.32.log)
:

**Comment by SpaiR** [comment]
In cases like that it's helpful to open up your maps in DM. If you would try that for your map you'd see this:

<details>
<summary>in DM</summary>
<img width="719" alt="Screenshot 2022-08-07 at 15 32 56" src="https://user-images.githubusercontent.com/12721311/183290756-fb8803c7-ac27-4ea0-b41f-f7f9db6436b8.png">
</details>

and this in the actual file:

<details>
<summary>in notepad</summary>
<img width="407" alt="Screenshot 2022-08-07 at 15 31 38" src="https://user-images.githubusercontent.com/12721311/183290770-f54d11f1-75d9-4bea-8357-906f67115509.png">
</details>

I'm not sure how it's possible to corrupt the map in that way from StrongDMM. There is basically a part of the map (only the part) which is not properly formatted. Seems like the problem during the invalid merge conflict resolving or else.

tl;dr You need to formatting for your map manually. You can try to use mapmerge2 tool, but I'm not sure if it won't do the worst.
